client/src/views/WeatherPage.vue

```diff
@@ -1,168 +1,38 @@
 <template>
-  <div id="app">
-    <main>
-      <div class="search-box">
-        <input
-          type="text"
-          class="search-bar"
-          placeholder="Search..."
-          v-model="query"
-          @keyup.enter="fetchWeather"
-        />
-      </div>
-      <div class="weather-wrap" v-if="typeof weather.main != 'undefined'">
-        <div class="location-box">
-          <div class="location">
-            {{ weather.name }}, {{ weather.sys.state }}
-            {{ weather.sys.country }}
-          </div>
-          <div class="date">{{ dateBuilder() }}</div>
-          <div class="weather-box">
-            <div class="temp">{{ Math.round(weather.main.temp) }} °F</div>
-            <div class="weather">{{ weather }}</div>
+  <div>
+      <div v-for="(weather, index) in weatherList" :key="weather.id">
+          <div v-if="index % 8  === 0">
+              in {{( index * 3)}} hours, it will be
+              {{weather.main.temp}} °F
           </div>
-        </div>
       </div>
-    </main>
   </div>
+  
 </template>
 
 <script>
 export default {
   data() {
-    return {
-      api_key: "8b81c25b4594c01f9e7d4caec4793f7a",
-      url_base: "https://api.openweathermap.org/data/2.5/",
-      query: "",
-      weather: {},
-    };
+      return {
+          api_key: '8b81c25b4594c01f9e7d4caec4793f7a',
+          url_base: 'https://api.openweathermap.org/data/2.5/',
+          lat: "-34.0522",
+          lon: "118.2437",
```

Comment:
for the purpose of consistency, our convention for shortening longitude in this codebase has been "lng," can we get that updated?

Reply:
Before the comment is made that OpenWeather uses 'lon' in its fetches/data, yes, however it is good to keep the variable's name consistent with the rest of the project despite the eccentricities of one API. Given it only appears in the request url, keeping naming consistency shouldn't cause any confusion.